packages/app-store/stripepayment/lib/PaymentService.ts

```diff
@@ -259,6 +257,16 @@ export class PaymentService implements IAbstractPaymentService {
         payment_method: setupIntent.payment_method as string,
         off_session: true,
         confirm: true,
+        metadata: this.generateMetadata({
+          bookingId,
+          userId: booking.user?.id,
+          username: booking.user?.username,
+          bookerName: booking.attendees[0].name,
+          bookerEmail: booking.attendees[0].email,
+          bookerPhoneNumber: booking.attendees[0].phoneNumber ?? null,
```

Comment:
attendees[0] could potentially be undefined. Would be a bug if that happens, but still possible.